remove debug label from documentation (#964)
asreview/asreview: docs/source/intro/datasets.rst

**Included** A binary variable indicating the existing labeling decisions with
``0`` = irrelevant/excluded, and ``1`` = relevant/included. Different column
names are allowed, see the table. It can be used for:

- **Screening**: In ASReview LAB, if labels are available for a part of the
  dataset (see :doc:`partly labeled data <../features/pre_screening>`), the
  labels will be automatically detected and used for prior knowledge. The first
  iteration of the model will then be based on these decisions and used to
  predict relevance scores for the unlabeled part of the data.
- **Exploration**: You can explore an existing fully labeled dataset in the Exploraton

>   Mode. The relevant records will be displayed in green during screening. 

Reviewer: @terrymyc can you rephrase/fix this?

> - **Exploration**: You can explore an existing fully labeled dataset in the Exploraton
>   Mode. The relevant records will be displayed in green during screening. 

Reviewer 2: ```suggestion
- **Exploration**: You can explore a completely labeled dataset in the Exploration
  Mode. The relevant/irrelevant label in the dataset will be displayed on each record.
```
